libres/lib/enkf/tests/enkf_obs_vector.cpp

```diff
@@ -28,41 +30,41 @@ void test_create(enkf_config_node_type *config_node) {
         obs_vector_alloc(SUMMARY_OBS, "OBS", config_node, 100);
     test_assert_true(obs_vector_is_instance(obs_vector));
     {
-        const int_vector_type *step_list = obs_vector_get_step_list(obs_vector);
+        const std::vector<int> step_list = obs_vector_get_step_list(obs_vector);
```

Comment:
```suggestion
        auto step_list = obs_vector_get_step_list(obs_vector);
```
or, more correctly,
```suggestion
        const auto& step_list = obs_vector_get_step_list(obs_vector);
```

It's style so and some people don't like `auto` (compile-time type inference), so up to you.